From 7a1f514feeb65ff8b903f07cff50a408748ed45a Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 12 Oct 2023 18:07:44 +0200
Subject: [PATCH 0672/1141] arm64: dts: st: add power domain in mdf node on
 stm32mp251

Add power domain for MDF on STM32MP251 device tree.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I4a350181a236e4b32bd9583dbc43ea1bd82b6d71
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/336790
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index af791375e0f6..177402265372 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1702,6 +1702,7 @@ mdf1: mdf@404d0000 {
 				resets = <&rcc MDF1_R>;
 				reset-names = "mdf";
 				feature-domains = <&rifsc STM32MP25_RIFSC_MDF1_ID>;
+				power-domains = <&RET_PD>;
 				status = "disabled";
 
 				sitf0: sitf@80 {
-- 
2.39.2

